GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -127,6 +127,38 @@ private static void UpdateGuiFocus(ToolBarSelectedButton currentTab)
             }
         }
 
+        private void OnGuiOverviewSelect()
+        {
+            var descriptionTextStyle = CreateDescriptionTextStyle();
+            EditorGUILayout.LabelField("Building Instant Game with Quick Deploy", EditorStyles.boldLabel);
+            EditorGUILayout.BeginVertical(UserInputGuiStyle);
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField("Building an instant game with Quick Deploy will significantly reduce the size" +
+                                       " of your game by storing the main AssetBundle of the game to a remote server.",
+                descriptionTextStyle);
+            EditorGUILayout.EndHorizontal();
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField("Use Quick Deploy features to Build and Deploy AssetBundle, as well as to " +
+                                       "Create a Loading Screen that will be displayed at game startup when the " +
+                                       "AssetBundle will be downloaded from a remote server. ", descriptionTextStyle);
+            EditorGUILayout.EndHorizontal();
+            EditorGUILayout.Space();
+            EditorGUILayout.Space();
+            EditorGUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField("Use GooglePlayInstant's Build Settings Window to change " +
```

Comment:
`"Use Google Play Instant's \"{0}\" window to customize the scenes included in the instant app. Then use the \"{1}\" menu option to test the instant app loading the AssetBundle from the remote server. Finally, select the \"{2}\" menu option to build the app in a manner suitable for publishing on Play Console."`

Reference existing strings for:
0 = Build Settings
1 = Build and Run
2 = Build for Play Console

Reply:
Resolved.

Wasn't sure what you meant by "Existing Strings" tho. Lmk if you had something else in mind than what i did.